### ISSUE TITLE
FreeRADIUS 3.0.22 (debian) ignores clients

### ISSUE THREAD

**Issue by mrvne** [FreeRADIUS/freeradius-server#4101]
Latest version (3.0.22) ignores clients that are being loaded from sql. I assume it is not reading the global clients list correctly.

```
rlm_sql (sql): Adding client X.X.X.X (client1) to global clients list
rlm_sql (X.X.X.X): Client "client1" (sql) added
...
...
Ignoring request to auth address X.X.X.X port 1812 from unknown client X.X.X.X port 39532 proto udp
Ignoring request to auth address X.X.X.X port 1812 from unknown client X.X.X.X port 56511 proto udp
```

Previous version (3.0.21) works fine.

**Comment by alandekok** [comment]
Already fixed in the v3.0.x branch.

**Comment by sveldhuisen** [comment]
Just a small note for anyone else who is havinf the same problem, but uses LDAP as a client backend. I was facing the exact same issue, but my clients are fetched from LDAP. The 3.0.x branch mentions a specific fix for loading clients via SQL:

`Fix (again) "read clients from SQL" functionality.`

I can confirm the fix also works for loading clients from an LDAP enabled directory service.
